bob-apps/bin/blob_recovery_tool/structs.rs

```diff
@@ -119,6 +134,21 @@ pub(crate) struct Record {
     pub data: Vec<u8>,
 }
 
+impl Record {
+    pub(crate) fn migrate(self, source: u32, target: u32) -> AnyResult<Self> {
+        match (source, target) {
+            (source, target) if source == target => Ok(self),
+            (1, 2) => self.mirgate_v1_to_v2(),
```

Comment:
This is probably an error: current BLOB version is '1' and previous is '0'

Reply:
Fixed